src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -125,10 +126,10 @@ Amplify.configure(myAppConfig);
 
 [question]: # (Do we want to address non-AppSync on this job or link to it? Or does this become an Expander section?)
 
```

Comment:
I missed asking this Q again in our Slack conversation but I made the choice to put this next section in an expander vs on the page. Does this work or do we want to take this option out of the expander and list AppSync as Option 1 and non-AppSync as Option 2?